cumulusci/cumulusci.yml

```
@@ -252,6 +252,10 @@ tasks:
         description: Retrieves a list of the currently available Permission Set License definition keys
         class_path: cumulusci.tasks.preflight.licenses.GetAvailablePermissionSetLicenses
         group: Salesforce Preflight Checks
+    get_existing_recordtypes:
```

```suggestion
    get_existing_record_types:
```

Suggesting a task name that I think fits `snake_case` a little better but I don't feel strongly about it.